dspace/config/modules/external-providers.cfg

```diff
@@ -25,4 +25,9 @@ vufind.url.search = https://vufind.org/advanced_demo/api/v1/search
 
 scielo.url = https://search.scielo.org/?output=ris&q=
 
+#################################################################
+#----------------------   Pubmed   -----------------------------#
+#---------------------------------------------------------------#
+pabmed.url.search = https://eutils.ncbi.nlm.nih.gov/entrez/eutils/esearch.fcgi
+pabmed.url.fetch = https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi
```

Comment:
Please fix the misspelling of Pubmed in these configs.  Should be `pubmed.url.*` obviously